pkg/k8s/apis/cilium.io/v2/client/register.go

```diff
@@ -95,13 +95,18 @@ func CreateCustomResourceDefinitions(clientset apiextensionsclient.Interface) er
 		return createIdentityCRD(clientset)
 	})
 
+	// Reviewers: I'm not sure if it makes sense to wrap this under the feature
+	// flag check since we have related Makefile changes.
```

Comment:
I don't think we necessarily need to avoid registering the CRD. There is no harm in registering a CRD that will never have a resource created for it. IMO, we can avoid clutter the codebase with a bunch of ifs and only keep it to a minimum where we truly need it.

Reply:
Also, I didn't quite understand why we are removing it from the Makefile too?

Reply:
Yes, I also feel that we don't necessarily have to avoid registering the CRD (hence the note in the description).  I haven't removed the CLRP related operation from the Makefile (just the CCLRP)?

Reply:
FWIW, I was also initially confused by the Clusterwide LRP changes in this PR since there's no dedicated commit for it and it's not called out in the PR description or commit message (hence unclear whether it's intentional).

Reply:
Cool, so looking over this thread again, I had the same confusion as Joe.

The conclusion here is that we will _not_ register the CLRP CRD conditionally, as there's no harm in it.